### PR TITLE
Fix publishing to crates.io; Release 0.2.2

### PR DESCRIPTION
**Pull request by marioiordanov**
## Development related changes

Move some files from `contracts` folder. 

## Checklist:

- [ ] Checked out the [contribution guidelines](CONTRIBUTING.md)
- [ ] Applied formatting - `./scripts/format.sh`
- [ ] No linter errors - `./scripts/clippy_check.sh`
- [ ] No unused dependencies - `./scripts/check_unused_deps.sh`
- [ ] No spelling errors - `./scripts/check_spelling.sh`
- [ ] Performed code self-review
- [ ] Rebased to the latest commit of the target branch (or merged it into my branch)
    -   Once you make the PR reviewable, please avoid force-pushing
- [ ] Updated the docs if needed - `./website/README.md`
- [ ] Linked the [issues](https://github.com/0xSpaceShard/starknet-devnet-rs/issues) resolvable by this PR - [linking info](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
- [ ] Updated the tests if needed; all passing - [execution info](https://github.com/0xSpaceShard/starknet-devnet-rs/blob/main/.github/CONTRIBUTING.md#test-execution)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -59,31 +59,41 @@ jobs:
...
-      - run:
-          name: Clippy check
-          command: ./scripts/clippy_check.sh
-      - run:
-          name: Format check
-          command: ./scripts/format_check.sh
-      - run:
-          name: Code spell check
-          command: ./scripts/check_spelling.sh
+      # - run:
```
Comment (FabijanC) [inline .circleci/config.yml:62]: Uncomment?

File: .circleci/config.yml [hunk 2]
```diff
@@ -59,31 +59,41 @@ jobs:
...
+      #     name: Unit tests
+      #     command: cargo test --lib --bins --no-fail-fast
+      # - run:
+      #    name: Integration tests
+      #    # fails if jobs not limited; read more in https://github.com/0xSpaceShard/starknet-devnet-rs/issues/378
+      #    command: cargo test --jobs 9 --test '*' --no-fail-fast
+      - when:
+          condition:
+            not:
+              equal: [ main, << pipeline.git.branch >> ]
```
Comment (FabijanC) [inline .circleci/config.yml:90]: Doesn't this add a lot of time to the workflow? Locally for me a single publish dry run takes several minutes. Wouldn't it make more sense to actually only run it on `main`?

**Review by FabijanC — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (marioiordanov) [reply .circleci/config.yml:90]: on main there will be dry run an publish. The idea is to not have changes into main that cannot be published to crates.io.

Comment (marioiordanov) [reply .circleci/config.yml:90]: Also will increase the time

**Review by FabijanC — APPROVED** [review]

**Pull request merged**